### ISSUE TITLE
Call Audio

### ISSUE THREAD

**Issue by shri360** [phhusson/treble_experimentations#661]
Hi,

i have asus zenfone lite l1 device.
everything is fine working but while calling audio is not working, stay blank is any way to fix pls suggest the solution.

**Comment by proganime1200** [comment]
Yes same I did recently call one of my contacts it stays blank I'm using redmi go

**Comment by phhusson** [comment]
sorry @proganime1200 you're speaking of a different issue.

@shri360 on which image? did you disable treble settings? They are required for in-call audio.

**Comment by proganime1200** [comment]
edited my comment same issiue

**Comment by shri360** [comment]
@phhusson  almost all pie gsi image, their no stable in-call audio. any patches for universal fix for permanent??
